### DISCUSSION TITLE
Running GraphRAG Locally

### DISCUSSION THREAD

**Discussion by tatsuya-mk1** [microsoft/graphrag#1485] [Q&A] [answered]
Hi,
I am trying to run GraphRAG locally using gemma2 and nomic-embed-text.
I found a past post on how to do this in a post below.
https://github.com/microsoft/graphrag/issues/345#issuecomment-2211819596

However, when I tried to do this, I found out that there are no more files such as 
"packages/graphrag/llm/openai/openai_embeddings_llm.py" nor "Lib/site-packages/graphrag/llm/openai/utils.py".

Can anybody help me find these .py files or, if there is a better way to run graphrag locally, it will help if anybody can teach me.
Thank you in advance!

**Comment by yxp008** [comment, marked as answer]
Me too.Version 0.9.0 does not have it, but versions 0.5.0 and earlier do,so you may be able to use it.

**Reply by tatsuya-mk1** [reply]
Thank you very much for answering. As you said, I located the .py that needs to be changed in version 0.3.6.

Hope we can do the same with fnllm too.
